scipy/_distributor_init.py

```diff
@@ -0,0 +1,19 @@
+""" Distributor init file
+
+Distributors: you can add custom code here to support particular distributions
+of scipy.
+
+For example, this is a good place to put any checks for hardware requirements.
+
+The scipy standard source distribution will not put code in this file, so you
+can safely replace this file with your own version.
```

Comment:
That makes sense, this file should be empty. If we're keeping the path manipulation (@carlk still disagrees) then I guess it can go in a patch that's applied to ``_distributor_init.py`` at the start of the Appveyor run.